workspace/python/ds-pack/2.1.0-base-python-3.10.13/TUNNELING.md

```diff
@@ -0,0 +1,67 @@
+# Tunneling with VS Code
+
+This README provides instructions for setting up and managing remote tunnels using this image, making it easier for you to access your workspace remotely within VS Code.
+
+## Initializing a Remote Tunnel
+
+To create a new remote tunnel for your workspace, you can use the provided bash script:
+
+```bash
+bash /vscode-tunnel/scripts/start.sh
+```
+
+Running this script initializes and starts a new tunnel in the background, which can be accessed remotely. To ensure its functionality, a brief authentication step using GitHub is required.
+
+After executing the above command, you will see the following message in the console:
+
+```
+To grant access to the server, please log into https://github.com/login/device and use code <access-code>
+```
+
+Follow the link provided and enter the `<access-code>` displayed in the console.
+
+That's it! Your new, secure tunnel should now be active and accessible at the following URL: `https://vscode.dev/tunnel/workspace-<workspace-id>-0`
+
+## Managing Tunnels
+
+### Restarting a Tunnel
+
+If you need to restart your tunnel for smooth operation, you can use the following command:
+
+```bash
+bash /vscode-tunnel/scripts/refresh.sh
```

Comment:
Can we update these commands please to `restart-tunnel`, etc?

Reply:
Done